picoquic/picoquic_internal.h

```diff
@@ -844,7 +853,8 @@ uint32_t picoquic_protect_packet(picoquic_cnx_t* cnx,
     picoquic_connection_id_t * local_cnxid,
     uint32_t length, uint32_t header_length,
     uint8_t* send_buffer, uint32_t send_buffer_max,
-    void * aead_context, void* pn_enc);
+    void * aead_context, void* pn_enc,
+    int contains_retrans);
 
```

Comment:
Same remark. Please find another way to carry loss indications. For example, it would be less intrusive to have a loss detected flag per path, that would be reset when your loss detected bit is sent, or something like that.